File: 005/_005.java
public class _005 {
    public static void main(String[] args) {
        int result = 1;
        for (int i = 1; i < 20; i++)
            result = lcm(result, i);
        System.out.println(result);
    }

    public static int lcm(int a, int b) {
        return a * b / gcd(a, b);
    }

    public static int gcd(int a, int b) {
        while (b != 0) {
            int c = b;
            b = a % b;
            a = c;
        }
        return a;
    }
}

File: 005/005.go
package main

import "fmt"

func gcd(a, b int) int {
	for b != 0 {
		a, b = b, a%b
	}
	return a
}

func lcm(a, b int) int {
	return a * b / gcd(a, b)
}

func main() {
	result := 1
	for i := 1; i < 20; i++ {
		result = lcm(result, i)
	}
	fmt.Println(result)
}
